misc/de-metas-common/de-metas-common-bpartner/src/main/java/de/metas/common/bpartner/v2/response/JsonResponseComposite.java

```diff
@@ -61,18 +61,23 @@ public class JsonResponseComposite
 	@JsonInclude(Include.NON_EMPTY)
 	List<JsonResponseContact> contacts;
 
+	@JsonInclude(Include.NON_EMPTY)
+	List<JsonResponseBPBankAccount> bankAccounts;
+
 	@Builder(toBuilder = true)
 	@JsonCreator
 	private JsonResponseComposite(
 			@JsonProperty("orgCode") @Nullable final String orgCode,
 			@JsonProperty("bpartner") @NonNull final JsonResponseBPartner bpartner,
 			@JsonProperty("locations") @Singular final List<JsonResponseLocation> locations,
-			@JsonProperty("contacts") @Singular final List<JsonResponseContact> contacts)
+			@JsonProperty("contacts") @Singular final List<JsonResponseContact> contacts,
+			@JsonProperty("bankAccounts") @Singular final List<JsonResponseBPBankAccount> bankAccounts)
 	{
 		this.orgCode = orgCode;
 		this.bpartner = bpartner;
 		this.locations = coalesce(locations, ImmutableList.of());
 		this.contacts = coalesce(contacts, ImmutableList.of());
+		this.bankAccounts = coalesce(bankAccounts, ImmutableList.of());
```

Comment:
if you use `@Singular` then u never get null

Reply:
true story: I CPed 😊